src/js/routes/jobs.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 /* eslint-enable no-unused-vars */
 import { IndexRoute, Redirect, Route } from "react-router";
 
-import JobsTabContainer from "#PLUGINS/jobs/src/js/JobsTabContainer";
+import JobsOverview from "#PLUGINS/jobs/src/js/JobsTabContainer";
```

Comment:
Why the renaming? This change seems to be unncessary

Reply:
slipped :( the new component is named like this.

Reply:
please change, so I can take a final look and approve